src/common/json_output.rs

```diff
@@ -76,8 +80,13 @@ pub struct DescribeOutput {
 
 impl DescribeOutput {
     /// Creates new describe output from available
-    pub fn new(describe_info: EnclaveDescribeInfo, build_info: Option<EnclaveBuildInfo>) -> Self {
+    pub fn new(
+        enclave_name: Option<String>,
```

Comment:
Why don't you include enclave_name in the EnclaveDescribeInfo struct?

Reply:
I avoided changing the returned type by the enclave for keeping the backwards compatibility. I will look for a way of refactoring to avoid confusion.